### ISSUE TITLE
synthetic.xml: Bad default value for testSql for sql.OracleClient type

### ISSUE THREAD

**Issue by oxyasis** [xebialabs-community/xld-database-connection-test-plugin#5]
In type sql.OracleClient, the character '$' in the default value of the property testSql should be escaped in "SELECT * FROM v$version;"

If not, the variable is interpreted by the shell and replaced by an empty value.
```
Uploading file checkOracleDB.sh to working directory.
Executing /tmp/ot-20180724T161207954/generic_plugin.tmp/checkOracleDB.sh on host Infrastructure/****
SELECT * FROM v;
SQL*Plus: Release 11.2.0.4.0 Production on Tue Jul 24 16:12:08 2018
Copyright (c) 1982, 2013, Oracle. All rights reserved.
Connected to:
Oracle Database 11g Enterprise Edition Release 11.2.0.4.0 - 64bit Production
SQL> SQL> SQL> SELECT * FROM v
*
ERROR at line 1:
ORA-00942: table or view does not exist
Disconnected from Oracle Database 11g Enterprise Edition Release 11.2.0.4.0 - 64bit Production
Execution failed with return code 1
```

With the value "SELECT * FROM v\\$version;", the check is successfully executed
```
Uploading file checkOracleDB.sh to working directory.
Executing /tmp/ot-20180724T162915995/generic_plugin.tmp/checkOracleDB.sh on host Infrastructure/****
SELECT * FROM v$version;
SQL*Plus: Release 11.2.0.4.0 Production on Tue Jul 24 16:29:16 2018
Copyright (c) 1982, 2013, Oracle. All rights reserved.
Connected to:
Oracle Database 11g Enterprise Edition Release 11.2.0.4.0 - 64bit Production
SQL> SQL> SQL>
BANNER
--------------------------------------------------------------------------------
Oracle Database 11g Enterprise Edition Release 11.2.0.4.0 - 64bit Production
PL/SQL Release 11.2.0.4.0 - Production
CORE 11.2.0.4.0 Production
TNS for IBM/AIX RISC System/6000: Version 11.2.0.4.0 - Production
NLSRTL Version 11.2.0.4.0 - Production
SQL> Disconnected from Oracle Database 11g Enterprise Edition Release 11.2.0.4.0 - 64bit Production
```

**Comment by jdewinne** [comment]
Sounds logic. Would you be willing to create a PR?

**Comment by oxyasis** [comment]
Done.

**Comment by jdewinne** [comment]
Thanks. Released in [v1.0.5](https://github.com/xebialabs-community/xld-database-connection-test-plugin/releases/tag/v1.0.5)
